packages/node-sdk/README.md

```diff
@@ -174,6 +223,25 @@ bucketClient.initialize().then({
   bucketClient.getFeature("invalid-feature") // feature doesn't exist
 })
 
+function deleteTodo(todoId: string) {
+  // get the feature information
+  const { isEnabled, config: { payload: confirmationConfig }} = bucketClient.getFeature("delete-todos")
+
+  // check that feature is enabled for user
+  if (!isEnabled) {
+    return;
+  }
+
+  // finally, check if we enabled the "confirmation" dialog for this user and only
+  // show it in that case.
+  // since we defined `ConfirmationConfig` as the only valid payload for `delete-todos`,
+  // we have type-safety helping us with the payload value.
+  if (confirmationConfig.shouldShowConfirmation) {
+    showMessage("Are you really sure you want to delete this ite,?");
+    // ... rest of the code
+  }
+}
+
```

Comment:
the image here below refers to line 223. I think it makes sense to separate this part about the configs into a separate code block. So: 

1. original code block
2. original image showing you the type error 
3. new code block showing how the configs are typed.

btw.:

>    showMessage("Are you really sure you want to delete this ite,?");

looks off